qiskit_ibm_runtime/runtime_job.py

```diff
@@ -222,29 +218,20 @@ def error_message(self) -> Optional[str]:
         self._set_status_and_error_message()
         return self._error_message
 
-    def wait_for_final_state(
-        self, timeout: Optional[float] = None, wait: float = 5
-    ) -> None:
+    def wait_for_final_state(self, timeout: Optional[float] = None) -> None:
         """Poll the job status until it progresses to a final state such as ``DONE`` or ``ERROR``.
 
         Args:
             timeout: Seconds to wait for the job. If ``None``, wait indefinitely.
-            wait: Seconds between queries.
 
         Raises:
             JobTimeoutError: If the job does not reach a final state before the
                 specified timeout.
```

Comment:
You removed the import but did not update the docstring. Should the code continue to raise a `JobTimeoutError` given that only an implementation detail (socket vs. polling) changed?